bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -307,7 +307,7 @@ spec:
                 - --enable-leader-election
                 - --zap-log-level=info
                 - --zap-time-encoding=rfc3339nano
-                image: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:0.68.0
```

Comment:
we dont have such tag and this is what I told you about the fact that they keep here a static version of the operator - maybe for a reason?